testing/src/test/java/io/stargate/it/cql/AuthenticationTest.java

```diff
@@ -0,0 +1,84 @@
+package io.stargate.it.cql;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.datastax.oss.driver.api.core.AllNodesFailedException;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.cql.Row;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.stargate.auth.model.AuthTokenResponse;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.http.models.Credentials;
+import io.stargate.it.storage.ClusterConnectionInfo;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+// TODO find a workaround to reenable this test
+@Disabled(
+    "Needs to run in isolation because it relies on a static variable set from a system property")
+public class AuthenticationTest extends JavaDriverTestBase {
+
+  public AuthenticationTest(ClusterConnectionInfo backend) {
+    super(backend);
+    enableAuth = true;
+  }
+
+  @BeforeAll
+  public static void beforeAll() {
+    System.setProperty("stargate.cql_use_auth_service", "true");
```

Comment:
I had to disable these tests because the value of this property is stored in a static variable, it will be already set if other tests have run first.
We should find a better way to handle config, system properties are really not flexible.

Reply:
Can you please make sure we have an issue for this?